src/deploy/AWSDeployer.js

```diff
@@ -380,6 +390,43 @@ class AWSDeployer extends BaseDeployer {
     this.log.info('parameters updated');
   }
 
+  async cleanUpBuckets() {
+    this.log.info('--: cleaning up stray temporary S3 buckets ...');
+    let res = await this._s3.send(new ListBucketsCommand({}));
+    const helixBuckets = res.Buckets.filter((b) => b.Name.startsWith('poly-func-maker-temp-'));
```

Comment:
~At some paint we should change the bucket name prefix.

Reply:
yes... and/or create a tag